src/shared/cards/Card Template.js

```diff
@@ -0,0 +1,73 @@
+import Card from 'cards/Card';
+import { Set } from 'immutable';
```

Comment:
Okay this is confusing but
`Set` is a built in native thing in JS so you don't need to import it
`Immutable.Set` is a different thing that I'm not using server-side any more, so just don't have this line basically